src/pages/all-on-one-v2/all-on-one-v2.ts

```diff
@@ -109,8 +125,19 @@ export class AllOnOneV2Page {
     return this.selectedManoeuvre === key;
   }
 
+  controlledStopTap() {
+    this.isControlledStopDone = true;
+    this.summaryMetaDataService.setControlledStopComplete();
```

Comment:
It looks like we're setting two things to true that mean the same thing, it's okay for now but they might become out of sync in the future and we won't know which is the correct one.